src/components/WebexActivity/WebexActivity.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {format, isToday, isSameWeek, isYesterday} from 'date-fns';
+
+import WebexAvatar from '../WebexAvatar/WebexAvatar';
+import {useActivity, usePerson} from '../hooks';
+
+import './WebexActivity.scss';
+
+/**
+ * Returns a formatted timestamp based on the given date's offset from the current time.
+ *
+ * @param {Date} timestamp A string value representing a date, specified in a format recognized by the Date.parse()
```

Comment:
I had this param description for before, when it was a string... now it's a date object 🤦🏻‍♀️
Maybe something like `Date instance to format` ??